src/mixins/filing-template-mixin.ts

```diff
@@ -335,23 +337,21 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
         isFutureEffective: false
       },
       registration: {
-        startDate: this.getRegistration.startDate,
-        nameRequest: {
-          legalType: this.getEntityType
+        business: {
+          identifier: this.getTempId
         },
         businessAddress: this.getRegistration.businessAddress,
-        // *** FUTURE: save businessType here
         contactPoint: {
           email: this.getBusinessContact.email,
           phone: this.getBusinessContact.phone,
           ...this.getBusinessContact.extension
             ? { extension: +this.getBusinessContact.extension }
             : {}
         },
+        naics: this.getRegistration.naics,
```

Comment:
naics should be inside business https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/example_data/schema_data.py#L1388

Reply:
You're right! Fixed in new commit.